### ISSUE TITLE
To what extent does resizing not working counts as a bug?

### ISSUE THREAD

**Issue by Incogdino** [nus-cs2103-AY2425S1/forum#694]
![image](https://github.com/user-attachments/assets/0274b984-a2b4-49ea-b36e-d5c53049897d)

What is the screen limitation when it comes to resizing of the application? For example, on a large/average sized monitor, the resizing may work to fix UI issues but when limited to a laptop with a smaller monitor size, the resizing may not fix the issue.

What warrants the case for it to be considered as a bug?

**Comment by damithc** [comment]
@Incogdino You can follow the minimum screens sizes given in https://nus-cs2103-ay2425s1.github.io/website/admin/tp-constraints.html#constraint-screen-resolution

**Comment by NorbertLoh** [comment]
> @Incogdino You can follow the minimum screens sizes given in https://nus-cs2103-ay2425s1.github.io/website/admin/tp-constraints.html#constraint-screen-resolution

Hi prof, does this mean that if a part of my UI gets truncated/overflowed even after resizing to the maximum, it is considered a bug? Thank you!

**Comment by damithc** [comment]
> Hi prof, does this mean that if a part of my UI gets truncated/overflowed even after resizing to the maximum, it is considered a bug? Thank you!

Yes, if the UI doesn't work fully in a screen size that it is expected to work in, it can be considered a bug.
